core/templates/dev/head/services/SiteAnalyticsService.js

```diff
@@ -196,6 +196,18 @@ oppia.factory('SiteAnalyticsService', ['$window', function($window) {
     registerStewardsLandingPageEvent: function(viewerType, buttonText) {
       _sendEventToGoogleAnalytics(
         'ClickButtonOnStewardsPage', 'click', viewerType + ':' + buttonText);
-    }
+    },
+    registerSaveRecordedAudioEvent: function(explorationId) {
+      _sendEventToGoogleAnalytics(
+        'SaveRecordedAudio', 'click', explorationId);
```

Comment:
can we move this to the line above?

Reply:
Done

Reply:
@suicide11, I was asking you to move `'SaveRecordedAudio', 'click', explorationId);` a line above (Following max 80 chars per line.). Sorry for the misunderstanding.

Reply:
Done